tests/testthat/test-utils_default_stats_formats_labels.R

```diff
@@ -199,22 +199,24 @@ testthat::test_that("labels_use_control works as expected", {
 })
 
 testthat::test_that("summary_formats works as expected", {
-  result <- summary_formats() %>%
-    unlist() # More compact fruition
+  result <- testthat::expect_warning(
+    summary_formats() %>%
+      unlist() # More compact fruition
+  )
   res <- testthat::expect_silent(result)
   testthat::expect_snapshot(res)
 
-  result <- summary_formats(type = "counts", include_pval = TRUE)
+  result <- testthat::expect_warning(summary_formats(type = "counts", include_pval = TRUE))
```

Comment:
I wonder if this will capture the warning or the result
```suggestion
testthat::expect_warning(result <- summary_formats(type = "counts", include_pval = TRUE))
```

Reply:
that is true!